community/megatools/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --disable-docs
+
+make
+
+./install-sh -c -d "$1/usr/bin"
+./install-sh -c -d "$1/usr/share/doc/megatools"
```

Comment:
This directory is created but nothing is ever copied to it?

Reply:
I forgot to remove that when i added the `--disable-docs` flag